Source/BugsnagKSCrashSysInfoParser.m

```diff
@@ -16,9 +16,10 @@
 #define PLATFORM_WORD_SIZE sizeof(void*)*8
 
 NSDictionary *BSGParseDevice(NSDictionary *report) {
-    NSMutableDictionary *device =
-    [[report valueForKeyPath:@"user.state.deviceState"] mutableCopy];
-    
+    NSMutableDictionary *device = [NSMutableDictionary new];
+    NSDictionary *state = [[report valueForKeyPath:@"user.state.deviceState"] mutableCopy];
```

Comment:
For consideration: Since `valueForKey:` can throw exceptions for incorrect keys, and since keypaths can't be checked by the compiler, is it worth considering wrapping these in a `@try`/`@catch`? It's certainly very unlikely these will throw, but not impossible.